content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -718,3 +718,71 @@ Clean up the Pod when you're finished with it:
 ```shell
 kubectl delete pod myapp
 ```
+
+### Custom Profile {#custom-profile}
+
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
+
+You can define a partial container spec as a custom profile in either YAML or JSON format.
+The custom profile can be applied to an ephemeral container or a debugging container in a copied Pod 
+or a debugging Pod using the `--custom` flag.
+
+{{< note >}}
+Custom profile only supports the modification of the ephemeral container or debugging container spec.
+It does not support the modification of the Pod spec.
+Modifications for `command`, `image`, `lifecycle`, `name` and `volumeDevices` fields 
+via a custom profile are not allowed.
```

Comment:
This paragraph doesn't make sense.
If modifications to a Pod spec is not supported, why are we talking about the specific fields then?